sssd_test_framework/roles/ad.py

```diff
@@ -1655,16 +1655,18 @@ def __init__(self, role: AD, name: str) -> None:
         self._search_base: str = f"cn=policies,cn=system,{self.role.host.naming_context}"
         """Group policy search base."""
 
-        self._dn = self.get("DistinguishedName")
+        self._dn = self._get("DistinguishedName")
         """Group policy dn."""
 
-        self._cn = self.get("CN")
+        self._cn = self._get("CN")
         """Group policy cn."""
 
-    def get(self, key: str) -> str | None:
+    def _get(self, key: str) -> str | None:
```

Comment:
Why did you make it private? And why _dn and _cn are private?

Reply:
I think this was an artifact when I tried to make GPOs inherit ADObject and I didn't think about it after it worked. I do recall that it wasn't overriding the method from ADObject.get, short answer, I don't remember, fixed.